src/main/resources/changelog/changes/v4.1/create_call_number_tables.xml

```diff
@@ -60,4 +60,30 @@
 
   </changeSet>
 
+  <changeSet id="MSEARCH-936@@add-last_updated_date-to-call-number" author="svitlana_kovalova1">
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="call_number"/>
+    </preConditions>
+
+    <addColumn tableName="call_number">
+      <column name="last_updated_date" type="DATETIME" defaultValueComputed="CURRENT_TIMESTAMP">
+        <constraints nullable="false"/>
+      </column>
+    </addColumn>
+  </changeSet>
```

Comment:
You can move it to call_number table creation changeset as this feature still in feature branch.